common/ferrostar/src/snapshots/ferrostar__simulation__tests__state_from_polyline.snap

```diff
@@ -8,7 +9,7 @@ current_location:
     lng: -149.543469
   horizontal_accuracy: 0
   course_over_ground:
-    degrees: 0
+    degrees: 288
```

Comment:
This reflects a bug in the previous implementation where negative bearings were being coalesced to 0 in [location_simulation_from_coordinates](https://github.com/stadiamaps/ferrostar/pull/328/files#diff-5668545870bc04900016eb8c61c7df2af165eb8dde8b4072d42396986d2d2e66L109)

Reply:
Good catch, and thanks for the explanation! That explains the same question I had on https://github.com/stadiamaps/ferrostar/pull/323#discussion_r1821766112 yesterday!